index.js

```diff
@@ -17,9 +15,8 @@ function getDown (defaults) {
   }
 }
 
-function getLocation (name) {
-  mkdirp.sync(tmpdir)
-  return path.join(tmpdir, name)
+function getLocation () {
+  return tempy.directory()
 }
 
 function clean (location) {
```

Comment:
I'm unsure if we should keep this or not. It doesn't really make sense now so should just pass in a noop function to `wrap()`?